lib/rdoc/generator/template/rails/class.rhtml

```diff
@@ -18,6 +18,7 @@
 </head>
 
 <body>
+    <a class="sr-only sr-only-focusable" href="#content"> Skip to content </a>
```

Comment:
```suggestion
    <a class="sr-only sr-only-focusable" href="#content" data-turbolinks="false"> Skip to content </a>
```

or `data-turbo="false"` if this is updated to turbo drive.

It works like a charm on https://deploy-preview-161--nervous-roentgen-6c1877.netlify.app/classes/RDoc/Generator/SDoc.html
It seems to do something weird on https://deploy-preview-161--nervous-roentgen-6c1877.netlify.app/classes/Object.html#content

I think it's because of turbolinks.